waspc/e2e-test/GoldenTest.hs

```diff
@@ -83,3 +99,24 @@ runGoldenTest goldenTest = do
     writeFileManifest baseAbsFp filePaths manifestAbsFp = do
       let sortedRelativeFilePaths = unlines . sort . map (makeRelative baseAbsFp) $ filePaths
       writeFile manifestAbsFp sortedRelativeFilePaths
+
+    -- This function normalizes all package.json files for comparison as `npm install` can overwrite
+    -- them when creating/updating package-lock.json. Different versions of npm may produce different
+    -- (but semantically equivalent) package.json files, thus triggering false positives.
+    -- Ref: https://github.com/wasp-lang/wasp/issues/482
+    reformatPackageJsonFiles :: [FilePath] -> IO ()
+    reformatPackageJsonFiles allOutputFilePaths = do
+      let packageJsonFilePathSuffix = FP.pathSeparator : "package.json"
+      let packageJsonFilePaths = filter (packageJsonFilePathSuffix `isSuffixOf`) allOutputFilePaths
+      mapM_ reformatJson packageJsonFilePaths
+      where
+        aesonPrettyConfig = Config {confIndent = Spaces 4, confCompare = compare, confNumFormat = Generic, confTrailingNewline = True}
```

Comment:
I think wasp uses 2 spaces when generating package.json, so maybe we should use that?

Reply:
Yep good catch, can do!